Assets/MixedRealityToolkit/Definitions/InputSystem/MixedRealityInputSystemProfile.cs

```diff
@@ -56,6 +56,16 @@ public SystemType RaycastProviderType
             internal set { raycastProviderType = value; }
         }
 
+        /// <summary>
+        /// Maximum number of colliders that can be detected in a SphereOverlap scene query.
+        /// </summary>
+        public int FocusQueryBufferSize => focusQueryBufferSize;
+
+        [SerializeField]
+        [Range(32, 2048)]
+        [Tooltip("Maximum number of colliders that can be detected in a SphereOverlap scene query.")]
+        private int focusQueryBufferSize = 128;
```

Comment:
Nit: Change the value in the FocusProvider.cs thing to 128 as well (or change this to 64) to be consistent. It doesn't make sense for us to have two different defaults.